app/jobs/create_account_fcrepo_job.rb

```diff
@@ -0,0 +1,6 @@
+# This job is the same as its parent, but chains to the next step
+class CreateAccountFcrepoJob < CreateFcrepoEndpointJob
```

Comment:
I think what cbeer is saying (and I agree with) is that we only need a single class (`CreateFcrepoEndpointJob`), we don't need two classes with an inheritance relationship as there is no other user of `CreateFcrepoEndpointJob`